### DISCUSSION TITLE
Any plan on using hyper backend with static-curl feature ?

### DISCUSSION THREAD

**Discussion by eburghar** [sagebind/isahc#353] [Questions] [answered]
https://aws.amazon.com/fr/blogs/opensource/how-using-hyper-in-curl-can-help-make-the-internet-safer/

**Comment by sagebind** [comment, marked as answer]
No specific plan at the moment, though it would be an upstream responsibility of [curl-rust](https://github.com/alexcrichton/curl-rust) to make this possible. Currently the hyper support in curl is only experimental AFAIK.
